x-pack/test/functional_with_es_ssl/apps/cases/edit_case_form.ts

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import uuid from 'uuid';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default ({ getPageObject, getService }: FtrProviderContext) => {
+  const common = getPageObject('common');
+  const header = getPageObject('header');
+  const testSubjects = getService('testSubjects');
+  const find = getService('find');
+  const cases = getService('cases');
+  const retry = getService('retry');
+  const comboBox = getService('comboBox');
+
+  describe('Edit case', () => {
```

Comment:
I think for all edits we should check also if the user action has been created and is correct. Or do you think is better to write another test suite only for user actions?